holidays/financial/ny_stock_exchange.py

```diff
@@ -181,7 +181,7 @@ def _populate(self, year):
                 begin + timedelta(days=n)
                 for n in range((end - begin).days + 1)
             ):
-                if self._is_weekend(d):
+                if d.weekday() in {SA.weekday, SU.weekday}:
```

Comment:
Why not `_is_weekend(d)`? Here and two changes below

Reply:
Just a merge error, fixed.